src/ci/docker/host-x86_64/dist-x86_64-linux/libgccjit.version

```diff
@@ -0,0 +1 @@
+b6f163f52
```

Comment:
This should be a full commit hash, not a partial prefix. I'm not sure what the value in keeping it in a separate file is either (vs. having it explicit above).

Reply:
This matches the release name in [my GCC fork](https://github.com/antoyo/gcc/releases). I can make it use the full hash if necessary.

Reply:
Mostly to prevent having it in multiple places as we clone the GCC repository in a few places. I'll use a full commit hash though.

Reply:
Well actually it's only used in this file, so not much point indeed. Removing this file.